src/deserialize/parse.rs

```diff
@@ -278,16 +278,43 @@ fn read_number(
 }
 
 fn read_char(chars: &mut std::iter::Enumerate<std::str::Chars<'_>>) -> Result<Edn, Error> {
-    let i = chars
+    let element = chars
         .clone()
-        .next()
-        .ok_or_else(|| Error::ParseEdn("Could not identify symbol index".to_string()))?
-        .0;
-    let c = chars.next();
-    c.ok_or(format!("{c:?} could not be parsed at char count {i}"))
-        .map(|c| c.1)
-        .map(Edn::Char)
-        .map_err(Error::ParseEdn)
+        .enumerate()
+        .take_while(|&(i, c)| i <= 200 && !c.1.is_whitespace())
+        .map(|(_, c)| c.1)
+        .collect::<String>();
+
+    let mut consume_chars = |n| {
+        // We need to map/collect to consume out of the Enumerate
+        let _ = chars.take(n).map(|c| c.1).collect::<String>();
+    };
+
+    match element {
```

Comment:
I'm sorry, I don't get this part, perhaps my Clojure skills are a bit rusty :drum: 

If I recall correctly, using `\a` is just like `'a'` (char) in Rust. Then, what do those `\\xyz` mean?

I was trying them (`\\newline`, `\\space`, etc) in the [REPL](https://tryclojure.org/), and only `\\newline` resolved to something.

![2023-12-15_09-32](https://github.com/naomijub/edn-rs/assets/15306309/7e60c3f5-f386-4c64-8c61-4e8797cda145)

Reply:
@Grinkers ?